dir.props

```diff
@@ -114,7 +114,7 @@
   <!-- Dependency auto-upgrade configuration -->
   <ItemGroup>
     <DependencyBuildInfo Include="CoreFx">
-      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/$(DependencyBranch)</RawUrl>
+      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/dev/api</RawUrl>
```

Comment:
Hmm, maybe I'm missing something with gitignore affecting merges (I've been under the impression gitignore only affects interactions with the working dir) but I don't see how that helps. Just to make sure we're on the same page, this is the scenario I'm seeing:
```
o <- Merge, to push to [master], depends on dev/api with a default clean merge
|\
| o <- [dev/api] changed to depend on dev/api
o | <- [master] depends on master
...
|/
o <- Common ancestor, depends on master
```
Where the problem is that merging dev/api into master brings along the change to depend on the dev/api buildinfo, but we want a dependency on master. (I'm actually curious if the master packages would be compatible with dev/api changes, and if we might want to depend on dev/api for a single build to "seed" the API changes into master? If so, the change I suggested above does that.)